.circleci/config.yml

```diff
@@ -95,6 +95,36 @@ jobs:
           command: npm test
 
 
+
       # Save test results for timing analysis
       - store_test_results:
           path: test_results
+
+  register_image:
+    docker:
+      - image: circleci/ruby:2.5.3-stretch-node
+    steps:
+      - setup_remote_docker:
+          version: 18.05.0-ce
+      - checkout
+      - run:
+          name: Build Image & Register Task
+          command: |
+            docker build -t suldlss/rialto-webapp:latest .
```

Comment:
won't this fail on https://github.com/sul-dlss/rialto-webapp/blob/master/Dockerfile#L2 ?